### ISSUE TITLE
[Feature] Consider add a route animation mode

### ISSUE THREAD

**Issue by xyhxx** [fluttercandies/flutter_wechat_assets_picker#107]
**Version information (版本信息) 5.1.3
 - Device: Iphone 11
 - OS: *e.g. iOS 14.4*
 - Package Version: *e.g. v2.12.0*
 - Flutter Version: *e.g. v2.0.4*

**Describe the solution you'd like (描述你期望的解决方案)**
I use CupertinoPageRoute for route management, and the page has exit animation, flitter_ wechat_ assets_ The picker enters from the bottom up, which leads to a black area on the exit page

我英语不太好，稍微补充一下，我的路由是用的CupertinoPageRoute，每个页面有个离场动画，flutter_wechat_assets_picker的入场动画是从底部向上展示的，这个会导致页面有个黑色区域，图片我会放在附加内容上。会不会考虑添加一个路由模式解决这个问题，我现在只能通过降低路由过渡时间解决这个问题。

**Additional context (附加内容)**

![image](https://user-images.githubusercontent.com/51938216/116639072-3b4ea680-a99a-11eb-98cc-a94d56edf33f.png)

**Comment by AlexV525** [comment]
Do you use GetX to integrate with your routes?

> 是否使用了 GetX 集成了路由？

Please provide a minimum reproducible demo.

> 提供一个最小化的 demo。

**Comment by xyhxx** [comment]
Yes i used Get

dependencies:
```
  get: ^4.1.4
  wechat_assets_picker: ^5.1.3
```

code:
```
import 'package:flutter/material.dart';
import 'package:get/get.dart';
import 'package:wechat_assets_picker/wechat_assets_picker.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return GetMaterialApp(
      home: MyAppHome(),
      getPages: [
        GetPage(name: '/test', page: () => Test()),
      ],
    );
  }
}

class MyAppHome extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(),
      body: Center(
        child: ElevatedButton(
          onPressed: () {
            Get.toNamed('/test');
          },
          child: Text('to Route'),
        ),
      ),
    );
  }
}

class Test extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(),
      body: Center(
        child: ElevatedButton(
          onPressed: () {
            AssetPicker.pickAssets(
              context,
              maxAssets: 4,
              routeDuration: Duration(seconds: 1),
            );
          },
          child: Text('choose Image'),
        ),
      ),
    );
  }
}


```

**Comment by AlexV525** [comment]
Fixed and will be published as 5.1.4 later in about 10 minutes. Thanks for provide this solid demo. 👍 

> 已修复，将在 5.1.4 中发布，约 10 分钟。感谢提供了有效的 demo~
